minio/minioadmin.py

```diff
@@ -322,6 +323,17 @@ def info(self) -> str:
         )
         return response.data.decode()
 
+    def account_info(self, prefix_usage: bool = False) -> str:
+        """Get usage information for the authenticating account"""
+        response = self._url_open(
+            "GET",
+            _COMMAND.ACCOUNT_INFO,
+            query_params={
```

Comment:
Send `query_params` only if `prefix_usage` is `true`

Reply:
Fixed, Also switched to Mapping in _url_open for query params to avoid further type errors